jetty-infinispan/src/main/java/org/eclipse/jetty/session/infinispan/InfinispanSessionDataStore.java

```diff
@@ -258,7 +259,7 @@ public boolean exists(String id) throws Exception
     {
         // TODO find a better way to do this that does not pull into memory the
         // whole session object
-        final AtomicReference<Boolean> reference = new AtomicReference<>();
+        final AtomicBoolean reference = new AtomicBoolean();
```

Comment:
This seems out of scope of the PR.

Reply:
Not really, as it changes the method signatures where this is used, so they can avoid boxing